Typescript error

- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Steps to reproduce
Just install and run typescript.

### What is expected?
No error.

### What is actually happening?
```sh
node_modules/antd/lib/mentions/index.d.ts:25:51 - error TS2307: Cannot find module '_rc-mentions@0.3.1@rc-mentions/lib/Option'.

25     static Option: React.FunctionComponent<import("_rc-mentions@0.3.1@rc-mentions/lib/Option").OptionProps>;
                                                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/antd/lib/mentions/index.d.ts:34:50 - error TS2307: Cannot find module '_rc-mentions@0.3.1@rc-mentions/lib/util'.

34     getFilterOption: () => false | typeof import("_rc-mentions@0.3.1@rc-mentions/lib/util").filterOption | undefined;
                                                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/antd/lib/slider/index.d.ts:2:22 - error TS7016: Could not find a declaration file for module 'rc-slider/lib/Slider'. '/Users/hsuting/Desktop/work/meep-lerna/node_modules/rc-slider/lib/Slider.js' implicitly has an 'any' type.
  Try `npm install @types/rc-slider` if it exists or add a new declaration (.d.ts) file containing `declare module 'rc-slider/lib/Slider';`

2 import RcSlider from 'rc-slider/lib/Slider';
```

| Environment | Info |
|---|---|
| antd | 3.19.0 |
| React | 16.8.6 |
| System | Mac os 10.14.4 |
| Browser | Chrome 74.0.3729.157 |


<!-- generated by ant-design-issue-helper. DO NOT REMOVE -->

--- a/components/mentions/__tests__/index.test.js
+++ b/components/mentions/__tests__/index.test.js
@@ -9,6 +9,10 @@ function simulateInput(wrapper, text = '', keyEvent) {
   const myKeyEvent = keyEvent || {
     which: lastChar.charCodeAt(0),
     key: lastChar,
+    target: {
+      value: text,
+      selectionStart: text.length,
+    },
   };
 
   wrapper.find('textarea').simulate('keyDown', myKeyEvent);

--- a/components/mentions/index.tsx
+++ b/components/mentions/index.tsx
@@ -4,10 +4,11 @@ import * as React from 'react';
 import { polyfill } from 'react-lifecycles-compat';
 import RcMentions from 'rc-mentions';
 import { MentionsProps as RcMentionsProps } from 'rc-mentions/lib/Mentions';
+import { OptionProps as RcOptionProps } from 'rc-mentions/lib/Option';
 import Spin from '../spin';
 import { ConfigConsumer, ConfigConsumerProps, RenderEmptyHandler } from '../config-provider';
 
-const { Option } = RcMentions;
+const Option: React.FunctionComponent<RcOptionProps> = RcMentions.Option;
 
 function loadingFilterOption() {
   return true;
@@ -119,7 +120,7 @@ class Mentions extends React.Component<MentionProps, MentionState> {
     return children;
   };
 
-  getFilterOption = () => {
+  getFilterOption = (): any => {
     const { filterOption, loading } = this.props;
     if (loading) {
       return loadingFilterOption;

